src/krux/pages/login.py

```diff
@@ -334,7 +321,7 @@ def _load_key_from_words(self, words, charset=LETTERS, new=False):
             )
             if not derivation_path:
                 derivation_path = key.derivation
-            wallet_info = key.fingerprint_hex_str(True) + "\n"
+            wallet_info = "\n"
```

Comment:
Please use offset instead of "\n" to change position of text

Reply:
Can't use offset here because the text uses ` info_box=True` on `draw_hcentered_text`. The box looks broken  on the screen

Reply:
Ok, maybe then `info_box=True` doesn't need to be set (and an extra rectangle doesn't need to be drawn) right below when drawing the fingerprint over it

Reply:
Yes, this can be done!